notebooks/merged-datasets.ipynb

```diff
@@ -0,0 +1,911 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2b924115-4ee1-48c9-afc1-eb7fab3646d6",
+   "metadata": {},
+   "source": [
+    "# Creating a database with artists from image filenames and cleaned art styles"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "58556723-6676-40bf-903a-a89e669cf276",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "45e11042-1efc-424e-b93e-094dc039d50c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.preprocessing import MultiLabelBinarizer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "c5846968-80c7-42da-a11c-bbb09c0a4e92",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.read_csv('../raw_data/artists.csv')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d7473d4a-4fd3-436b-b205-655d079e2ca1",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "## 1. Creating broader art style labels"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "6c7a7e92-e89c-4bdc-bbc9-0c086504c47e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "genre_mapping = {\n",
+    "    \"Expressionism\": \"Expressionism\",\n",
+    "    \"Expressionism,Abstractionism\": \"Expressionism,Abstractionism\",\n",
+    "    \"Social Realism,Muralism\": \"Social Realism,Muralism\",\n",
+    "    \"Impressionism\": \"Impressionism\",\n",
+    "    \"Surrealism,Impressionism\": \"Surrealism,Impressionism\",\n",
+    "    \"Surrealism\": \"Surrealism\",\n",
+    "    \"Realism,Impressionism\": \"Realism,Impressionism\",\n",
+    "    \"Byzantine Art\": \"Byzantine Art\",\n",
+    "    \"Post-Impressionism\": \"Impressionism\",\n",
+    "    \"Symbolism,Art Nouveau\": \"Symbolism,Art Nouveau\",\n",
+    "    \"Northern Renaissance\": \"Renaissance\",\n",
+    "    \"Suprematism\": \"Suprematism\",\n",
+    "    \"Symbolism\": \"Symbolism\",\n",
+    "    \"Cubism\": \"Cubism\",\n",
+    "    \"Baroque\": \"Baroque\",\n",
+    "    \"Romanticism\": \"Romanticism\",\n",
+    "    \"Primitivism,Surrealism\": \"Primitivism,Surrealism\",\n",
+    "    \"Mannerism\": \"Mannerism\",\n",
+    "    \"Primitivism\": \"Primitivism\",\n",
+    "    \"Proto Renaissance\": \"Renaissance\",\n",
+    "    \"Early Renaissance\": \"Renaissance\",\n",
+    "    \"High Renaissance\": \"Renaissance\",\n",
+    "    \"Impressionism,Post-Impressionism\": \"Impressionism\",\n",
+    "    \"High Renaissance,Mannerism\": \"High Renaissance,Mannerism\",\n",
```

Comment:
Elsewhere we map say "Northern Renaissance" to "Renaissance" ... here should we then map "High Renaissance,Mannerism" to "Renaissance,Mannerism"?